avalon/pipeline.py

```diff
@@ -673,6 +675,7 @@ def create(name, asset, family, options=None, data=None):
         plugins.append(plugin)
 
     assert plugins, "No Creator plug-ins were run, this is a bug"
+    print("Here: %s" % instance)
```

Comment:
@mottosso Did you intend to commit this?

Reply:
:S

Reply:
I'll remove it now.